Sample Data/Custom/Netclean_Incidents_CL.json

```diff
@@ -0,0 +1,2162 @@
+[
+    {
+      "TenantId": "5035c591-e16a-410b-8162-a184e4e9c185",
+      "SourceSystem": "RestAPI",
+      "MG": "",
+      "ManagementGroupName": "",
+      "TimeGenerated [UTC]": "3/21/2023, 9:54:34.306 AM",
+      "Computer": "",
+      "RawData": "",
+      "countOfAllNearByFiles_s": 8,
+      "sha1_s": "9dd32ac721317d5b8122f8e729dd1cdcaba25629",
+      "m365filePath_s": "",
+      "m365WebUrl_s": "",
+      "m365CreatedBymail_s": "",
+      "m365LastModifiedByMail_s": "",
+      "m365LibraryId_s": "",
+      "m365LibraryDisplayName_s": "",
+      "m365Librarytype_s": "",
+      "m365siteid_s": "",
+      "m365sitedisplayName_s": "",
+      "m365sitename_s": "",
+      "filePath_s": "C:\\Users\\Administrator\\Downloads\\test-images\\img2.jpg",
+      "agentType_s": "computer",
+      "nearbyFiles_sha1_s": "9dd32ac721317d5b8122f8e729dd1cdcaba25629,f81bb3cd3cf07934a48cca4e855039f969ec9ef6,1d32c57f7130bdd80be9e4566381627dfd3ef3fe,e173d7ee8648bdfcca20cbcfc0688ea61e76276b,",
+      "Identifier_s": "",
+      "foundTime_s": "",
+      "creationTime_s": "",
+      "lastAccessTime_s": "",
+      "lastWriteTime_s": "",
+      "agentInformatonIdentifier_s": 1,
+      "Agentidentifier_s": "",
+      "triggerSource_s": "",
+      "domain_s": "BUILTIN",
+      "domainName_s": "",
+      "hasCollectedNearbyFiles_s": "True",
```

Comment:
@NCsteven, this field should be **"hasCollectedNearbyFiles_b": True,**  as it is a Boolean value, thanks.

Reply:
Hi the logic app interprets the incoming webhook value as a string with  True or False, not 1 or 0. i would have to update sample data, workbook and the LogicAPP that recives  the weebhook inorder to change this.
 For now it can only be true or false, but since our incidents come in parts where all new information is updated, it could  change over time. Please explain why this is a requirement and if deemed necessery.